From: Richard Acayan <mailingradian@gmail.com>
Date: Thu, 3 Aug 2023 18:53:45 -0400
Subject: [PATCH] dt-bindings: arm: qcom: document google,bonito-sdc board

The google,bonito-sdc board compatible represents the Google Pixel 3a XL
with a SOFEF00 panel. Document it in the bindings.

Signed-off-by: Richard Acayan <mailingradian@gmail.com>
---
 Documentation/devicetree/bindings/arm/qcom.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/arm/qcom.yaml b/Documentation/devicetree/bindings/arm/qcom.yaml
index 9679fed..4dce0d8 100644
--- a/Documentation/devicetree/bindings/arm/qcom.yaml
+++ b/Documentation/devicetree/bindings/arm/qcom.yaml
@@ -865,6 +865,7 @@ properties:
 
       - items:
           - enum:
+              - google,bonito-sdc
               - google,sargo
           - const: qcom,sdm670
 
